reflex/state.py

```diff
@@ -2471,6 +2473,7 @@ def create(cls, *children, **props) -> "Component":
         setattr(reflex.istate.dynamic, state_cls_name, component_state)
         component = component_state.get_component(*children, **props)
         component.State = component_state
+        component = into_component(component)
```

Comment:
i think we want to invert the order of these lines; the `.State` should always be attached to the thing that's returned from `.create(...)`